apollo/plugin.py

```diff
@@ -0,0 +1,173 @@
+from datetime import datetime
+from dataclasses import dataclass
+import httpx
+from typing import Optional, List
+
+from lutraai.augmented_request_client import AugmentedTransport
+
+
+@dataclass
+class EmploymentHistory:
+    organization_name: Optional[str]
+    start_date: Optional[datetime]
+    end_date: Optional[datetime]
+    title: Optional[str]
+
+
+@dataclass
+class FundingEvent:
+    date: datetime
+    news_url: Optional[str]
+    type: str
+    investors: List[str]
+    amount: str
+    currency: str
+
+
+@dataclass
+class Organization:
+    name: str
+    website_url: Optional[str]
+    blog_url: Optional[str]
+    angellist_url: Optional[str]
+    twitter_url: Optional[str]
+    facebook_url: Optional[str]
+    phone: Optional[str]
+    founding_year: Optional[int]
+    industries: List[str]
+    keywords: List[str]
+    estimated_num_employees: Optional[int]
+    street_address: Optional[str]
+    city: Optional[str]
+    state: Optional[str]
+    country: Optional[str]
+    seo_description: Optional[str]
+    annual_revenue: Optional[float]
+    funding_events: List[FundingEvent]
+
+
+@dataclass
+class PeopleEnrichmentData:
```

Comment:
nit: avoid using generic terms like "Data" in the name.

this looks like just a person's profile?

ApolloPersonProfile?